config/routes.rb

```diff
@@ -11,38 +11,42 @@
         get 'find', to: 'find#show'
         get 'find_all', to: 'find#index'
         get 'random', to: 'random#show'
+        get 'most_revenue', to: 'revenue#index'
       end
 
       namespace :invoice_items do
         get 'find', to: 'find#show'
         get 'find_all', to: 'find#index'
         get 'random', to: 'random#show'
       end
-      
+
       namespace :merchants do
-         get '/find', to: 'find#show'
-         get '/find_all', to: 'find#index'
-         get '/random', to: 'random#show'
-       end
-       
+        get '/most_revenue', to: 'revenue#index'
+        get '/find', to: 'find#show'
+        get '/find_all', to: 'find#index'
+        get '/random', to: 'random#show'
+        get '/:id/favorite_customer', to: 'customers#show'
+        get '/:id/revenue', to: 'revenue#show'
```

Comment:
@samanthagongora I think that we need to have this point to it's own controller because we need the show functionality in the revenue controller to return all the revenue for a specific merchant by default or allow the date to go through as params for just that merchant. Since this is an total sum of All merchants revenue on a date, I think it should be in it's own spot. What do you think?